projects/kit/components/slider/slider.component.ts

```diff
@@ -109,6 +109,7 @@ export class TuiSliderComponent {
         @Inject(TUI_SLIDER_OPTIONS) readonly options: TuiSliderOptions,
         @Inject(ElementRef) readonly elementRef: ElementRef<HTMLInputElement>,
         @Inject(USER_AGENT) private readonly userAgent: string,
+        @Inject(TUI_SLIDER_HAS_KEY_STEPS) private readonly hasKeySteps: boolean,
```

Comment:
can we just optional inject directive without extra token?

Reply:
Unfortunately, it will cause cyclic imports :((

https://github.com/Tinkoff/taiga-ui/blob/e35eea87afcb15935e780bca504dab785dc6d36d/projects/kit/components/slider/helpers/slider-key-steps.directive.ts#L32

Reply:
I tried even with `forwardRef(...)`
```ts
       @Inject(forwardRef(() => TuiSliderKeyStepsDirective)) private readonly hasKeySteps: unknown,
```

Reply:
```
core.js:6241 ERROR Error: Circular dep for TuiSliderComponent
    at getNodeInjectable (core.js:5981:1)
    at searchTokensOnInjector (core.js:5908:1)
    at getOrCreateInjectable (core.js:5812:1)
    at Module.ɵɵdirectiveInject (core.js:21157:1)
    at NodeInjectorFactory.TuiSliderKeyStepsDirective_Factory [as factory] (slider-key-steps.directive.ts:87:27)
    at getNodeInjectable (core.js:5993:1)
    at searchTokensOnInjector (core.js:5908:1)
    at getOrCreateInjectable (core.js:5812:1)
    at Module.ɵɵdirectiveInject (core.js:21157:1)
    at NodeInjectorFactory.TuiSliderComponent_Factory [as factory] 
```

Reply:
oh, got it ( sorry

Reply:
I have an idea
```ts
@Inject(Injector) private readonly injector: Injector,
//...
this.injector.get(TuiSliderKeyStepsDirective, null)
```

Give me a moment to try it :D

Reply:
@vladimirpotekhin I did not believe in it... but it works!

I will double check everything and push new commit later.

Reply:
wow cool 😎

Reply:
fixed

I've checked that it 
✅ works in `main`-branch (angular 9)
✅ works in `v3.x`-branch (angular 12)